comps/cores/mega/http_service.py

```diff
@@ -72,7 +72,9 @@ def _create_app(self):
         )
         async def _health_check():
             """Get the health status of this GenAI microservice."""
-            return {"Service Title": self.title, "Service Description": self.description}
+            from comps.version import __version__
+
+            return {"Service Title": self.title, "Version": __version__, "Service Description": self.description}
```

Comment:
Maybe version number could also be output to service log when service starts (e.g. from init method in here)?

Reply:
@eero-t  
it is good idea to output in the init function. 
This PR assumes that users mainly use HTTP request to communicate with microservice instead of looking into log for debugging. this PR focus on runtime HTTP request instead of debug logs.  maybe leave the log info in future PR?